xml
CopyEdit
<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationForm</class>
 <widget class="QDialog" name="RegistrationForm">
  <property name="windowTitle">
   <string>Регистрация происшествий</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="state1">
      <layout class="QFormLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelIncidentType">
         <property name="text"><string>Тип происшествия:</string></property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboIncidentType">
         <item><property name="text"><string>Бедствие</string></property></item>
         <item><property name="text"><string>Тренировка</string></property></item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelTime">
         <property name="text"><string>Дата и время (UTC):</string></property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDateTimeEdit" name="dateTimeIncident">
         <property name="calendarPopup"><bool>true</bool></property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelLocation">
         <property name="text"><string>Местоположение:</string></property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="editLocation"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel">
         <property name="text"><string>Широта:</string></property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="editLat"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel">
         <property name="text"><string>Долгота:</string></property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="editLon"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="state2">
      <layout class="QFormLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelVesselName">
         <property name="text"><string>Название судна:</string></property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="editVesselName"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelIMO">
         <property name="text"><string>IMO:</string></property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="editIMONumber"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="state3">
      <layout class="QVBoxLayout">
       <item>
        <widget class="QLabel" name="labelDescription">
         <property name="text"><string>Описание ситуации:</string></property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textDescription"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="state4">
      <layout class="QVBoxLayout">
       <item>
        <widget class="QLabel" name="labelConfirm">
         <property name="text"><string>Подтверждение информации:</string></property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="plainConfirm">
         <property name="readOnly"><bool>true</bool></property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="buttonPrev">
       <property name="text"><string>Назад</string></property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonNext">
       <property name="text"><string>Далее</string></property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonFinish">
       <property name="text"><string>Завершить</string></property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
